docker-compose.yml

```
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  nginx:
+    image: nginx:latest
+    ports:
+      - "80:80"
```

Éventuellement créer un ptit .env pour pouvoir faire ce genre de réglage (genre mon port 80 est déjà occupé sur mon poste)
```dotenv
NGINX_PORT=8000
```
et dans ton docker-compose.yml

```suggestion
      - ${NGINX_PORT:-8000}:80
```